.github/workflows/build-and-test-differential.yaml

```
@@ -22,6 +22,10 @@ on:
         required: false
         default: ""
         type: string
+      build-pre-command:
+        required: false
+        default: nice -n 19
```

```suggestion
        default: taskset --cpu-list 0-6
```

---

On a second look, this is to be used by the non-cuda build, so it's better to keep it empty.
```suggestion
        default: ""
```

---

I just kept it same so that non-cuda build run just like before, but I'm okay with keeping it empty. (I think it will still work fine)
Do you want me to change it?

---

Still, better to send it empty. I will create a PR to make the default on the action side empty too.

---

- Created https://github.com/autowarefoundation/autoware-github-actions/pull/336 to fix the default too.

---

fixed in https://github.com/autowarefoundation/autoware.universe/pull/10198/commits/376227b7aec03cba5d2ba156dd56b8bed1574f2e